contribute/en/Release-Process.md

```diff
@@ -236,15 +236,15 @@ Upload all the files. Then in the SourceForge web UI, for each of the files uplo
 
```

Comment:
Why is this change in this PR?

Reply:
I think it's marked as a change against next-release on my branch, which was this commit behind, but next-release here already has it, so won't change anything.

Reply:
That sounds wrong.  You should be basing your PR off the HEAD of your repo, which should be up to date with the HEAD of jamulussoftware:next-release.  There's shouldn't be any other commits involved.